### PR TITLE
Fix cli installer

### PR DESCRIPTION
**Pull request by schoren**
This PR...

## Changes

-

## Fixes

-

## Checklist

- [ ] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

### CODE REVIEW

File: cli/installer/kubernetes.go [hunk 1]
```diff
@@ -77,7 +77,7 @@ func kubernetesInstaller(config configuration, ui cliUI.UI) {
 
 func installCollector(config configuration, ui cliUI.UI) {
 	execCmd(
-		kubectlNamespaceCmd(config, "create -f "+collectorYaml),
+		kubectlNamespaceCmd(config, "apply -f "+collectorYaml),
```
Comment (schoren) [inline cli/installer/kubernetes.go:80]: this fixes a panic when trying to reinstall

File: cli/installer/kubernetes.go [hunk 2]
```diff
@@ -144,10 +144,16 @@ func fixTracetestConfiguration(conf configuration, ui cliUI.UI) {
...
+	ttp := createTmpFile("tracetest-provisioning", string(p), ui)
+	defer os.Remove(ttp.Name())
+
 	execCmd(
 		kubectlNamespaceCmd(conf,
-			"create configmap tracetest --from-file="+ttc.Name()+" -o yaml --dry-run=client",
-			"| sed 's#"+path.Base(ttc.Name())+"#config.yaml#' |",
+			"create configmap tracetest --from-file="+ttc.Name()+" --from-file="+ttp.Name()+" -o yaml --dry-run=client",
+			"| sed 's#"+path.Base(ttc.Name())+"#config.yaml#'",
+			"| sed 's#"+path.Base(ttp.Name())+"#provisioning.yaml#' |",
```
Comment (schoren) [inline cli/installer/kubernetes.go:156]: updates provisioning

File: cli/installer/templates/config.yaml.tpl [hunk 3]
```diff
@@ -5,22 +5,3 @@ postgres:
...
-  enabled: [{{ .enabledDemos }}]
-  endpoints:
-    pokeshopHttp: {{ .pokeshopHttp }}
-    pokeshopGrpc: {{ .pokeshopGrpc }}
-    otelFrontend: {{ .otelFrontend }}
-    otelProductCatalog: {{ .otelProductCatalog }}
-    otelCart: {{ .otelCart }}
-    otelCheckout: {{ .otelCheckout }}
-
-experimentalFeatures: []
```
Comment (schoren) [inline cli/installer/templates/config.yaml.tpl:26]: remove old config

File: cli/installer/templates/provision.yaml.tpl [hunk 4]
```diff
@@ -1,4 +1,7 @@
-dataStore:
+{{ if eq .installBackend "true" }}---
```
Comment (schoren) [inline cli/installer/templates/provision.yaml.tpl:1]: use new provision format

**Review by mathnogueira — APPROVED** [review]

**Pull request merged**